spyder_unittest/tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+"""
+Configuration file for Pytest.
+
+This contains the necessary definitions to make the main_window fixture
+available for integration tests.
+"""
+
+# Standard library imports
+import os
+
+# Third-party imports
+import pytest
+
+# This needs to be before any Spyder imports
+os.environ['SPYDER_PYTEST'] = 'True'
+
+# Spyder imports
+from spyder.app.tests.conftest import main_window
```

Comment:
We don't package our tests as part of the final package to save space. That's why I mentioned in my previous comment that you need to create your own `main_window` fixture here.

Reply:
Thanks, I missed that. You also suggested that I simplify the fixture, which sounds a bit scary because there are some places where it is not immediately clear to me what is going on, but I will have a go at it.